scss/settings/_settings.scss

```diff
@@ -800,7 +800,6 @@ $tab-active-color: $primary-color;
 $tab-item-font-size: rem-calc(12);
 $tab-item-background-hover: $white;
 $tab-item-padding: 1.25rem 1.5rem;
-$tab-expand-max: 6;
```

Comment:
No need to remove variables in `settings/_settings.scss` as it is automatically generated ;)

Reply:
Mh, in the src / scss directory? Should I revert this file change then?